examples/COCO/scripts/run16auto.batch

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Move this `scripts/` folder to somewhere else.

Actually, I'd suggest you @TarzanZhao  open another PR to contain these scripts and a README instruction on how to run Alpa on slurm clusters. This would be a great contribution.

Reply:
try to mimic this readme: https://github.com/alpa-projects/alpa/tree/main/examples/imagenet  by @merrymercy

Reply:
No problem